www/privoxy/Portfile

```diff
@@ -310,7 +334,8 @@ variant https_inspection \
                 -pass file:private/passphrase.txt
 
             # RSA
-            # openssl genrsa -aes256 -out private/ca.key.pem \\
+            # openssl genpkey -out private/ca.key.pem -algorithm RSA \\
+            #     -pkeyopt rsa_keygen_bits:2048 -aes256 \\
             #     -passout file:private/passphrase.txt
```

Comment:
I'm not clear on what this is documenting? Is it purely for the maintainer? If it is for the end user, it probably is better to explain it in a message on package install.

Reply:
It's just an obvious choice between `EC` and `RSA` certificates, and should be self-explanatory to anyone managing their PKI. The default here is EC, but the RSA option is there if anyone wants it. I think this falls into the category "if you have to ask…".

Reply:
Even sophisticated users don't read portfiles. I'd do another pull with some instructions in the install messages. Not strictly necessary of course.